src/scss/_color-use-cases.scss

```diff
@@ -1,8 +1,3 @@
-////
-/// @group o-message
-/// @link http://registry.origami.ft.com/components/o-message
-////
-
 /// Set error status colors
 @include oColorsSetColor('black-crimson-12.5', oColorsMix('black', 'crimson', 12.5)); //text
```

Comment:
I think we should kill custom colours within components in favour of using `oColorsMix('black', 'crimson', 12.5)` directly in brand config. What do you think? Because custom colours become a public interface and a breaking change to remove/stop using.